src/coins.h

```diff
@@ -285,8 +285,8 @@ class CCoinsViewCache : public CCoinsViewBacked
      * Note that lightweight clients may not know anything besides the hash of previous transactions,
      * so may not be able to calculate this.
      *
-     * @param[in] tx	transaction for which we are checking input total
-     * @return	Sum of value of all inputs (scriptSigs)
+     * @param[in] tx    transaction for which we are checking input total
+     * @return  Sum of value of all inputs (scriptSigs)
```

Comment:
for reviewers: I replaced tabs with spaces because if this block is moved, the `test/lint/lint-whitespace.sh` will fail.